aws/common/athena.tf

```
@@ -31,3 +31,15 @@ resource "aws_athena_workgroup" "primary" {
     CostCenter = "notification-canada-ca-${var.env}"
   }
 }
+
+resource "aws_athena_named_query" "create_table_alb_log" {
+  name      = "alb_log_create_table"
```

```suggestion
  name      = "create_table_alb_logs"
```
To match with the ID and table name